2 - RTB/Documentazione esterna/Analisi dei Requisiti/Analisi dei Requisiti.typ

```diff
@@ -306,29 +303,49 @@ $bold("Postcondizioni: ")$
 
 $bold("Scenario: ")$
 - l'utente avvia la modifica dell'ambiente di lavoro;
-- l'utente immette i dati richiesti.
+- l'utente regola le dimensioni dell'ambiente di lavoro a piacimento.
 
 $bold("Estensioni: ")$
-- UC-2.1 Visualizzazione errore dimensioni magazzino troppo piccole;
-- UC-5.1 Dimensioni negative o uguali a zero.
+- UC-2.1 Errore dimensioni magazzino troppo piccole;
+- UC-2.2 Errore dimensioni troppo piccole rispetto rispetto agli elementi nell'ambiente.
+
+== Errore dimensioni magazzino troppo piccole
+
+$bold("Descrizione: ")$
+l'utente vuole modificare le dimensioni dell'ambiente rimpicciolendo troppo le dimensioni dell'ambiente vuoto.
+
+$bold("Attore: ")$
+utente.
+
+$bold("Precondizioni: ")$
+- l'utente ha creato l' ambiente di lavoro manualmente;
+- l'ambiente è stato creato correttamente;
+- l'ambiente di lavoro risulta vuoto.
+
+$bold("Postcondizioni: ")$
+- all'utente viene notificato l'errore relativo al fatto che l'ambiente non può diventare più piccolo.
+
+$bold("Scenario: ")$
+- l'utente vuole ridurre le dimensioni dell'ambiente oltre una soglia minima.
 
-== Visualizzazione errore dimensioni magazzino troppo piccole
+== Errore dimensioni troppo piccole rispetto rispetto agli elementi nell'ambiente
 
 $bold("Descrizione: ")$
-i dati inseriti per la modifica dell'ambiente di lavoro generano conflitti con quanto configurato precedentemente.
+l'utente vuole modificare le dimensioni dell'ambiente rimpicciolendo troppo le dimensioni dell'ambiente non vuoto.
```

Comment:
ampliare il contesto: "Dato un ambiente già creato con già elementi posizionati (come scaffali e/o bin), l'utente cerca di ridurre le dimensioni dell'ambiente in modo eccessivo, non permettendo di mantenere gli elementi precedentemente posizionati". Riformula in modo più preciso.